src/modules/flow/http-server/http-server.c

```diff
@@ -212,8 +212,9 @@ common_response_cb(void *data, struct sol_http_request *request)
             SOL_INT_CHECK_GOTO(r, < 0, end);
             break;
         case SOL_HTTP_PARAM_HEADER:
-            if (streq(value->value.key_value.key, HTTP_HEADER_ACCEPT)) {
-                if (strstr(value->value.key_value.value, HTTP_HEADER_CONTENT_TYPE_JSON))
+            if (sol_str_slice_str_eq(value->value.key_value.key, HTTP_HEADER_ACCEPT)) {
+                if (memmem(value->value.key_value.value.data, value->value.key_value.value.len,
```

Comment:
use str slice functions, you can keep HTTP_HEADER_CONTENT_TYPE_JSON as a static const slice.